runtime/src/main/java/com/vaadin/quarkus/QuarkusVaadinServletService.java

```diff
@@ -103,10 +104,10 @@ public Optional<Instantiator> loadInstantiators() throws ServiceException {
             throw new ServiceException("Cannot init VaadinService "
                     + "because no CDI instantiator bean found.");
         }
-        final Bean<Instantiator> bean;
+        final Bean<InstantiatorFactory> bean;
         try {
             // noinspection unchecked
-            bean = (Bean<Instantiator>) beanManager.resolve(beans);
+            bean = (Bean<InstantiatorFactory>) beanManager.resolve(beans);
```

Comment:
the `beans` set seems to be for `Instantiator` instead of `InstantiatorFactory` (line 101)

Reply:
Good catch, thanks